### ISSUE TITLE
Flag stylistic sets defined with broken name table references as an error 

### ISSUE THREAD

**Issue by marekjez86** [googlei18n/nototools#61]
this is a tools portion of googlei18n/noto-alpha#241 

Syriac has stylistic sets defined with broken name table references. This need to be detected by a link tool. 

Imported from [Google Code issue #241](https://code.google.com/a/google.com/p/noto-alpha/issues/detail?id=241) created by [roozbeh@google.com](https://code.google.com/u/roozbeh@google.com/) on 2014-06-10T02:51:04.000Z:

----
The Syriac fonts appear to define stylistic sets with name reference IDs that don't exist in the font.

For example, feature #&nbsp;22 (ss01) in NotoSansSyriacEstrangela-Regular has a UINameID value of 256, but there is no such name record in the name table.

Here's a TTX dump of the relevant part of the feature definition.

```xml
<FeatureRecord index="22">
  <FeatureTag value="ss01"/>
  <Feature>
    <FeatureParamsStylisticSet>
      <Version value="0"/>
      <UINameID value="256"/>
    </FeatureParamsStylisticSet>
    <!-- LookupCount=1 -->
    <LookupListIndex index="0" value="31"/>
  </Feature>
</FeatureRecord>
```

**Comment by jungshik** [comment]
I guess what @marekjez86 meant is that link tools need to flag this case as an error.

**Comment by dougfelt** [comment]
Fixed in commit 0e292535b464d298fa86a52bce91c19f08400c47
